pkg/controller/chaosengine/chaosengine_controller.go

```diff
@@ -313,7 +357,8 @@ func newMonitorServiceForCR(cr *litmuschaosv1alpha1.ChaosEngine) (*corev1.Servic
 		WithPorts(getMonitoringENV()).
 		WithSelectorsNew(
 			map[string]string{
-				"app": cr.Name,
+				"app":  cr.Name,
+				"type": "exporter",
```

Comment:
What's the effect, If we deploy two chaosEngine in the same namespace, Then the two different services will be able to connect with exporter pod?

Reply:
This is a case we need to consider & test